### DISCUSSION TITLE
What to do with the records only seen by one screener?

### DISCUSSION THREAD

**Discussion by Woensvriend** [asreview/asreview#1716] [Q&A] [answered]
Hi all,

I'm about to start screening with ASReview and have discussed pros and cons with a more experienced colleague and ASReview user. One question came forward:

It seems that there's hiatus what to do with the records only seen by one screener. In the ideal situation, this set of these records will all be labelled as irrelevant. Can these indeed be excluded as irrelevant, albeit only assessed once? If so, what about the situation that there are relevant records in this set? I'm speculating, but it seems to me that a subphase is warranted, in which both screeners go through each other's records. This way, they can make sure that both have assessed the same records. Alternatively, a predefined low to medium interrater agreement can be used to prompt this additional subphase.

Here, I assumed that this set of papers is only discussed at the end of the screening. However, as indicated by Boetje & Van de Schoot (2024): all disagreements should be solved before starting the next phase, thus a subphase seems especially important for phase 1.

Is there anyone who has explored this before or has any experience with this? Should all records that have be screened only once, be checked by a second screener? Are reviewers of journals keen to point out that records were excluded after being assessed by only one screener?

Thanks in advance!

**Comment by FelixWdm** [comment]
Hi @Woensvriend , in the ideal scenario all records are seen by two independent researchers. If you're planning to save time using ASReview, you'll allmost always have records 'unseen' or when screening with two people only seen by one of them. To speed up the comparison, and focus on the records the reviewers disagree about, you can use the ASReview Add-on: [Link](https://github.com/langtonhugh/asreview_irr/tree/main).  I hope this helps.

**Reply by Woensvriend** [reply]
Thank you for your reply, FelixWdm. I think the ambiguity is in this sentence in the assumptions section of the article by [Boetje & Van de Schoot (2024)](https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-024-02502-7): "Also, it is expected two screeners will independently screen the records (...). Any disagreements should be solved before starting the next phase." I'm not sure how to interpret this. Do they suggest solving differences between relevant and irrelevant records at the end of each SAFE procedure phase?

**Reply by FelixWdm** [reply, marked as answer]
Good point, I would interpret a disagreement as a record seen by two screeners and both have made a different choice. @Rensvandeschoot right?
